scss/_patterns_links.scss

```diff
@@ -11,6 +11,40 @@
     &--no-underline {
       border: 0;
     }
+
+    &--soft {
+      color: $color-dark;
+
+      &:hover {
+        color: $color-accent;
+      }
+    }
+
+    &--strong {
+      border-bottom-color: $color-accent;
+      color: $color-dark;
+      font-weight: 400;
+
+      &:hover {
+        color: $color-accent;
+      }
```

Comment:
You need to repeat the `&:visited` styles here because `.p-link--strong:hover` has higher specificity than `a:visited`.

![vanilla-links-styles](https://cloud.githubusercontent.com/assets/1413534/22420026/1ccd5f8c-e6d8-11e6-8d46-f925b4c9f24b.png)